code/controllers/controllers.js

```diff
@@ -59,4 +62,61 @@ exports.pushArticleVotes = (req, res, next) => {
     .catch((err) => {
       next(err);
     });
-};
\ No newline at end of file
+};
+
+exports.postComment = (req, res, next) => {
+  //Firstly, need to check of article exists before trying to insert a comment fo that article
+  selectOneArticle(req)
+    .then((result) => {
```

Comment:
SQL will throw an error if we try to insert a comment with an article ID that doesn't exist.

Reply:
sorted via the other changes